python/tests/tornado/test_hyperbahn.py

```diff
@@ -54,11 +54,3 @@ def test_request():
             arg3='boo',
             headers={'as': 'qux'},
         )
-
-
-@pytest.mark.gen_test
-def test_register():
```

Comment:
We don't `yield hyperbahn.advertise` any longer. When user calls advertise, it just returns.